sections/semantics-grouping-content.include

```diff
@@ -1226,12 +1226,16 @@
     </dd>
   </dl>
 
-  The <{dl}> element <a>represents</a> a description list of zero or more <dfn>term-description groups</dfn>.
+    The <{dl}> element <a>represents</a> a description list of zero or more <dfn>term-description
+  groups</dfn>.
```

Comment:
Please keep the `dfn` element on a single line, so people can search the source for it.